### PR TITLE
feat: add image block

### PR DESCRIPTION
**Pull request by Walkeezy**
Add basic setup for image block: https://app.clickup.com/t/867766k8t

### CODE REVIEW

File: packages/image-block/manifest.json [hunk 1]
```diff
@@ -0,0 +1,3 @@
+{
+    "appId": ""
```
Comment (triggertoo) [inline packages/image-block/manifest.json:2]: could you already ask luca to create an app within the marketplace, so we can deploy the block for review?

Comment (Walkeezy) [reply packages/image-block/manifest.json]: I've created an app and pushed a first release: https://product.frontify.com/marketplace/app/cldju6bkp0001a8w43x1fgzzu?mode=view

File: packages/image-block/src/ImageBlock.tsx [hunk 2]
```diff
@@ -0,0 +1,18 @@
...
+import { useGuidelineDesignTokens } from '@frontify/guideline-blocks-shared';
+import 'tailwindcss/tailwind.css';
+import { Settings } from './types';
+
+export const ImageBlock = ({ appBridge }: BlockProps) => {
+    const isEditing = useEditorState(appBridge);
+    const [blockSettings] = useBlockSettings<Settings>(appBridge);
+    const { designTokens } = useGuidelineDesignTokens();
+
+    console.log({ isEditing, blockSettings, designTokens });
```
[the remark this answers is not in the document]
Comment (Walkeezy) [reply packages/image-block/src/ImageBlock.tsx:15]: Will do as soon as PR is ready :)

[the remark this answers is not in the document]
Comment (Walkeezy) [reply packages/image-block/src/ImageBlock.tsx]: Removed

**Review by fulopdaniel — APPROVED** [review]

**Comment by fulopdaniel** [comment]
👍 ⚡

**Review by triggertoo — APPROVED** [review]

**Pull request merged**